### DISCUSSION TITLE
Movies not showing in "Recently Added" section

### DISCUSSION THREAD

**Discussion by Nathan3690** [sct/overseerr#2780] [Help]
Hello,

I have two movies libraries : One for FHD and one for 4K.

When I add movies to the 4K library, they do not show up in overseer in the recently added section.

Only movies in FHD library show up.

Any ideas ?

All my Plex libraries are sync with Overseer.

Thank you

**Comment by Nathan3690** [comment]
Update : 

I just merged both libraries in Plex, rescan Plex but still no movies showing in Recently Added in Overseerr. Looks like that when the movie is only available in 4K it does not show up.

**Comment by stevendarby** [comment]
@Nathan3690 this issue was raised and closed a while back: https://github.com/sct/overseerr/issues/3010

Perhaps a few extra voices and suggestions could get it re-opened and implemented?
